src/configs/V1CNC_SkrPro_DualLR_2209_WiFi

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+set -e
+
+restore_configs
+
+export V1_VERSION=${V1_VERSION}D
+
+$CFGDIR/common/v1-base-config
+$CFGDIR/common/cnc-config
+$CFGDIR/boards/skr_pro
+$CFGDIR/accessories/dummy-extruder
+$CFGDIR/accessories/tmc2209
+$CFGDIR/accessories/reprap_discount_full_graphic_lcd
+$CFGDIR/accessories/dual-drivers-on-yz
+$CFGDIR/accessories/laser-32bit
+
+# Due to current limitation of Marlin, only two serial ports can be defined.
+# Using this config will swap out USB serial in favor of the an ESP3D module.
+opt_set SERIAL_PORT_2 6
+opt_set X_DRIVER_TYPE  "TMC2209"
```

Comment:
Do you need these? They should be set in the TMC2209 config allready, right?

Reply:
I used the V1CNC_SkrPro_DualLR_2209 as a template. I left the files completely intact and added the one line. But I think you are right, they should not be needed as they are already defined in the tmc2209 accessory config.

Reply:
Oh, that's weird. I must have compared it to the wrong file. I think the 2209 script must have changed after this script was written. I think it used to only call out the specific ports it was using.

It's up to you if you want to change it.